hera_qm/ant_metrics.py

```diff
@@ -431,3 +436,155 @@ def save_antenna_metrics(self, metricsJSONFilename):
 
         with open(metricsJSONFilename, 'w') as outfile:
             json.dump(allMetricsData, outfile, indent=4)
+
+# code for running ant_metrics on a file
+def get_metrics_OptionParser(method_name):
```

Comment:
I agree, they do seem generally useful. I'm personally not a huge fan of hiding large chunks of code in `__init__.py`, so I'll make a new utils module.